docs/VnVPlan/VnVPlan.tex

```diff
@@ -392,9 +393,44 @@ \subsubsection{Module ?}
 
 \end{enumerate}
 
-\subsubsection{Module ?}
+\subsubsection{Usability and Humanity Requirements}
+\begin{enumerate}
 
-...
+	\item{NFRT-UH-1\\}
+
+	Type: Dynamic, Manual
+
+	Initial State: Device is connected to the internet and application is not open
+
+	Input/Condition: The user launches the application on Google Chrome
+
+	Output/Result: The system can be assessed through Google Chrome
+
+	How test will be performed: The tester will attempt to launch the application on the Google Chrome
+	web browser and using a device that is connected to the internet.
+
+	\item{NFRT-UH-2\\}
+
+	Type: Dynamic, Manual
+
+	Initial State: Device is connected to the internet and the application is open
+
+	Input/Condition: User disconnects from the internet
+
+	Output/Result: The system notifies the user that there is no network connection and functionality
+	will be limited.
+
+	How test will be performed: The tester disconnect from the internet while the application is open.
```

Comment:
This one looks like it is referencing multiple requirements. Can you leave a comment for which ones? (Mainly for me when I create the traceability matrix. For label I would go with the first requirement if fulfills (i.e., NFRT-UH2).

Reply:
Added a comment for each non-functional requirement test indication which requirement it maps to